openwisp_utils/admin_theme/email.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import render_to_string
+from django.utils.html import strip_tags
+
+from . import settings as app_settings
+
+
+def send_email(subject, body, recipients, extra_context={}):
+
+    mail = EmailMultiAlternatives(
+        subject=subject,
+        body=strip_tags(body),
+        from_email=settings.DEFAULT_FROM_EMAIL,
+        to=recipients,
+    )
+    context = dict(
+        subject=subject, message=body, logo_url=app_settings.OPENWISP_EMAIL_LOGO,
+    )
+    context.update(extra_context)
```

Comment:
please move these lines in the if below, because it doesn't make sense to execute these if `app_settings.OPENWISP_HTML_EMAIL` is disabled